src/components/common/Toast/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { View } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { useToast } from 'react-native-toast-notifications';
+
+import { isIos } from '@/constants/platform';
+import { Colors } from '@/constants/theme';
+import Close from '@/icons/material/Close.svg';
+import ErrorIcon from '@/icons/svg/ErrorIcon.svg';
+import InfoIcon from '@/icons/svg/InfoIcon.svg';
+import SuccessIcon from '@/icons/svg/SuccessIcon.svg';
+
+import Text from '../Text';
+import Touchable from '../Touchable';
+import styles from './styles';
+
+export enum ToastTypes {
+  success = 'success',
+  error = 'error',
+  info = 'info',
+}
+
+export interface ToastProps {
+  title: string;
+  message: string;
+  type: 'success' | 'error' | 'info';
+  id: string;
+}
+
+export const toastProps = {
+  placement: 'top',
+  offset: isIos ? 55 : 35,
+  renderToast: (toast: any) => <Toast id={toast.id} {...toast.data} />,
+};
+
+const getToastStyles = (type: ToastTypes) => {
+  switch (type) {
+    case ToastTypes.success:
+      return { Icon: SuccessIcon, colors: ['#4A763B', '#54AA46'] };
```

Comment:
we could move the color arrays to the themes file

Reply:
U really think is worth it? I don't

Reply:
Just to maintain colors there